pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.black]
+line-length = 79
+target-version = ['py36', 'py37', 'py38']
```

Comment:
I just realized that this is a bit weird: for better or worse (mostly worse) the target version is py27.

Reply:
Black doesn't support or work with Python 2 at all. So `target-version` is a list of just the Python 3 versions it is targeting.

Reply:
To be clear, that doesn't mean that Python 2 support of the code needs to be dropped. Just that to actually _run_ Black you need a Python 3 runtime.

Reply:
Interesting, I guess as long as black doesn't have a tendency to make changes that are more than aesthetic in python 2 this is fine.

Reply:
[Black does introduce trailing commas](https://test-black.readthedocs.io/en/style-guide/style_guide/trailing_commas.html) in some tuples or lists, which will be viewed as invalid syntax in Python 2. However, this is not the case for `pandamonium`, and so it is unaffected.

FWIW, I have never had this break code for me.

Reply:
I don't even think that's a syntax error in python 2:
```
> python2 -c 'l=[1,2,3,]; t=(3,4,5,); print(l,t)'
([1, 2, 3], (3, 4, 5))
```

Reply:
Huh. Yeah, I guess I'm wrong on where this causes problems. I _have_ seen examples of Black breaking working Python 2 code, but apparently I don't remember how and this is hard to do.